### PR TITLE
[skdb] Thin ts client for remote SKDB.

### PR DESCRIPTION
**Pull request by beauby**
API:
- `let skdb = connect(endpoint, db, creds, schemas)`,
- `skdb.subscribe(table_name, init_cb, update_cb)`,
- `skdb.getTable(table_name)`.

Currently missing: writes (TBD with Daniel).

### CODE REVIEW

File: skdb-ts-thin/eslint.config.mjs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        { enforce: [{ prev: "*", next: "method", blankLine: "always" }] },
+      ],
+      "@stylistic/js/no-trailing-spaces": "error",
+      "jsdoc/require-jsdoc": ["warn", { publicOnly: true }],
+      "jsdoc/require-description": "warn",
+      "jsdoc/require-param-description": "warn",
+      "jsdoc/no-types": "warn",
+    },
+  },
+  { files: ["**/*.{mjs,ts}"], ...tseslint.configs.disableTypeChecked },
```
Comment (mbouaziz) [inline skdb-ts-thin/eslint.config.mjs]:
```suggestion
  { files: ["**/*.{mjs,js}"], ...tseslint.configs.disableTypeChecked },
```
Did you mean js?

File: skdb-ts-thin/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "chai": "^5.1.1",
+    "eslint": "^9.10.0",
+    "globals": "^15.9.0",
+    "mocha": "^10.7.3",
+    "ts-node": "^10.9.2",
+    "typescript-eslint": "^8.4.0"
+  },
+  "dependencies": {
+    "@stylistic/eslint-plugin-js": "^2.7.2",
+    "eslint-plugin-jsdoc": "^50.2.2",
```
Comment (mbouaziz) [inline skdb-ts-thin/package.json]: Aren't these two devDependencies only?

File: skdb-ts-thin/eslint.config.mjs
Comment (jberdine) [inline skdb-ts-thin/eslint.config.mjs:1]: Do we want to have just one eslint.config in the repo and add symlinks to it from the various dirs where eslint needs to find them?

Comment (beauby) [reply skdb-ts-thin/eslint.config.mjs:1]: I'm in favor, although we would probably need to ditch the `no-unused-vars` linter (or at least make it act upon `.js` files only), as it chokes on ts `enum`s (the `@typescript-eslint/no-unused-vars` is fine though).

**Comment by beauby** [comment]
Superseded by 09a8a6975e0da917ed3de1962ec5cf38bcf5c6c0.

**Pull request closed without merging**